Dockerfile

```diff
@@ -241,7 +241,19 @@ RUN curl -sSL https://get.haskellstack.org/ | bash -s -- -d /usr/bin
 
 #------------------------------------------------------------------------
 # REPO / ANDROID SDK
-FROM build AS androidbuild
+FROM ort-base-image AS androidbuild
+
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    sudo apt-get update -qq \
+    && DEBIAN_FRONTEND=noninteractive sudo apt-get install -y --no-install-recommends \
+    unzip \
+    && sudo rm -rf /var/lib/apt/lists/*
+
+RUN curl -ksS https://storage.googleapis.com/git-repo-downloads/repo | sudo tee /usr/bin/repo > /dev/null 2>&1 \
```

Comment:
Why is  Git-Repo now installed in `androidbuild`?  (I know that Git-Repo is used for development of AOSP, but is this the reason to install it in the `androidBuild`?

Reply:
Good catch. It's a bug. Fixing

Reply:
Is it supposed to be fixed already or will you force push again?

Reply:
Was fixed.